js-pkg/react/src/main.tsx

```diff
@@ -161,57 +161,70 @@ export function YDocProvider(props: YDocProviderProps) {
   return <YjsContext.Provider value={ctx}>{children}</YjsContext.Provider>
 }
 
-function useRedraw() {
-  const [_, setRedraw] = useState(0)
-  return useCallback(() => setRedraw((x) => x + 1), [setRedraw])
+function useVersion(): [number, () => void] {
+  const [version, setRedraw] = useState(0)
+  return [version, useCallback(() => setRedraw((x) => x + 1), [setRedraw])]
 }
 
-export type ObserverKind = 'deep' | 'shallow' | 'none'
+export type ObserverKind = 'deep' | 'shallow' | 'none' | boolean
```

Comment:
Hmm, what's the upside here? Is this solving a specific problem? If it's just another way to specify `deep` and `none`, I'd vote for removing the boolean. I think it adds ambiguity for readers - makes them have to know what `true` and `false` mean as configuration values alongside strings. It's also potentially confusing for writers who see that there are 5 possible values for this property and either have to look up the difference between them all, or, worse, intuit incorrectly what they mean.

Reply:
good point, removed